packages/feedback/components/feedback/src/feedback.scss

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright IBM Corp. 2023, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+$css--plex: true !default;
+
+@use '../../../../../globals/scss/vars' as *;
+@use '@carbon/styles/scss/theme' as *;
+
+:host(#{$c4ai-prefix}-feedback) hr {
+  border: 0;
+  background: var(--cds-border-subtle);
+  block-size: 1px;
+  margin-block: 1rem;
+}
+
+:host(#{$c4ai-prefix}-feedback) cds-slug .secondary {
+  color: var(--cds-text-secondary, #525252);
```

Comment:
you can use the carbon tokens directly

`color: $text-secondary;`

Reply:
Done